src/modules/apps/publish.ts

```diff
@@ -122,10 +122,7 @@ const publisher = (workspace: string = 'master') => {
             spinner.succeed(`${appId} was published successfully!`)
         }
       } catch (e) {
-        spinner.fail(`Fail to publish ${appId}`)
-        log.error(e.message)
```

Comment:
Are we sure we want to remove this `log.error(e.message)`?

Example: the 'minimum toolbelt version' error message - would the user be able to see that he/she had an obsolete `toolbelt` version when `publish`ing an app?

Reply:
Ah, nevermind... I saw the `builder-hub` PR now...